lib/index.js

```diff
@@ -63,12 +65,17 @@ Sentiment.prototype.analyze = function (phrase, opts, callback) {
     // Iterate over tokens
     var i = tokens.length;
     while (i--) {
-        var obj = tokens[i];
+        var obj = spelling.getSpellCheckedAfinnWord(labels, tokens[i]);
         if (!labels.hasOwnProperty(obj)) continue;
+
+        // Check for negation
```

Comment:
When I originally forked the repo, here we used to check if the previous word was a negation and invert the score. Now seems like we deal with negation further down. So we end up negating the score twice. Might be that my change to deal with negative words is redundant now